bftengine/src/bftengine/messages/CheckpointMsg.hpp

```diff
@@ -47,6 +51,10 @@ class CheckpointMsg : public MessageBase {
   void sign();
 
   void setSenderId(NodeIdType id) { sender_ = id; }
+  static bool equivalent(CheckpointMsg* a, CheckpointMsg* b) {
```

Comment:
What about the epoch number?

Reply:
you are right, I added only what I added :-)